doc/howtos/building-images.rst

```diff
@@ -257,6 +285,68 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
```

Comment:
Yes, the text should say "example ``ostro-image-noswupd`` is defined such that its default
 content corresponds to ``ostro-image-swupd``".